src/menu/src/Menu.js

```diff
@@ -26,7 +28,8 @@ const Menu = memo(function Menu(props) {
       : []
 
     if (menuItems.current.length === 0) {
-      throw new Error('The menu has no menu items')
+      setIsEmpty(true)
+      // throw new Error('The menu has no menu items')
```

Comment:
Can we remove this commented out code?

Reply:
Ya ya sure.

Reply:
I have committed the code without useState hook. Kindly review it.